third_party/gpus/cuda_configure.bzl

```diff
@@ -194,13 +194,13 @@ def _cuda_version(repository_ctx, cuda_toolkit_path, cpu_value):
     auto_configure_fail(
         "CUDA version detected from nvcc (%s) does not match " +
         "TF_CUDA_VERSION (%s)" % (version, environ_version))
-  return version
+  return ("64_" + version.replace(".", "")) if cpu_value == "Windows" else version
```

Comment:
To increase readability, could you replace ternary operator with an if else block?